docs/fields.md

```diff
@@ -180,9 +180,9 @@ Workflow is the definition of a workflow resource
 
 - [`input-artifact-s3.yaml`](https://github.com/argoproj/argo-workflows/blob/master/examples/input-artifact-s3.yaml)
 
-- [`k8s-jobs.yaml`](https://github.com/argoproj/argo-workflows/blob/master/examples/k8s-jobs.yaml)
+- [`k8s-jobs-log-selector.yaml`](https://github.com/argoproj/argo-workflows/blob/master/examples/k8s-jobs-log-selector.yaml)
```

Comment:
Maybe `k8s-resource-log-selector.yaml`? To me this is more about a feature of the "resource template" than about running jobs.

Reply:
Sounds good.